xbmc/GUITextureJobManager.h

```diff
@@ -0,0 +1,70 @@
+/*
+ *  Copyright (C) 2024 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#pragma once
+
+#include "GUILargeTextureManager.h"
+#include "threads/CriticalSection.h"
+#include "threads/Thread.h"
+
+#include <deque>
+
+using namespace std::chrono_literals;
+
+namespace KODI::GUILIB
+{
+
+class CGUITextureJobManager;
+class CGUITextureLoaderThread : protected CThread
+{
+public:
+  CGUITextureLoaderThread(CGUITextureJobManager* manager, unsigned int threadID);
+
+protected:
+  void Process() override;
+
+private:
+  static constexpr auto THREAD_SLEEP_DURATION{16ms};
```

Comment:
I would not use auto here, because to make it absolutely clear to the code reader what is used here exactly.